src/components/Header/index.tsx

```diff
@@ -4,10 +4,34 @@ import { ThemeToggler } from 'gatsby-plugin-dark-mode';
 import logoLight from '../../images/logos/nodejs-logo-light-mode.svg';
 import logoDark from '../../images/logos/nodejs-logo-dark-mode.svg';
 import { useMediaQuery } from '../../hooks/useMediaQuery';
+import LinkHeader from './LinkHeader'
+import AnchorHeader from './AnchorHeader'
+import { headersLinks } from './utils/links'
+import { HeaderType } from './utils/types'
 
 const Header = (): JSX.Element => {
   const isMobile = useMediaQuery('(max-width: 870px)');
-
+  const generateHeaderLinks = () => {
+    return headersLinks.map((header, idx) => {
```

Comment:
I think links should be used as props. the component should not be coupled to the specific links